.github/workflows/build_wheels.yml

```diff
@@ -271,13 +273,15 @@ jobs:
       if: startsWith(matrix.os_python.os, 'macos')
       run: brew install coreutils
     - name: Add checksums
+      if: ${{ needs.build_source.outputs.is_rc == 0 }}
       working-directory: apache-beam-source/wheelhouse/
       run: |
         for file in *.whl; do
           sha512sum $file > ${file}.sha512
         done
       shell: bash
     - name: Upload wheels as artifacts
+      if: ${{ needs.build_source.outputs.is_rc == 0 }}
```

Comment:
This does have one downside: if people are building their own beam version using this wheel, they won't get any cherry picks to the release branch.

I think that is fine though, and this seems worth it